src/content-url-resolver.ts

```diff
@@ -106,15 +106,15 @@ resolvers.push(async function landResolver(asset, options) {
 
 function defaultContentServerForNetwork(network: string, options: ResolversOptions) {
```

Comment:
Does not seem to be used anywhere on the whole org https://github.com/search?q=org:decentraland%20defaultContentServerForNetwork&type=code